.changes/v3.0.0/588-removals.md

```diff
@@ -0,0 +1,5 @@
+* Removed deprecated attributes `external_networks, default_gateway_network` from `vcd_edgegateway` [GH-588]
+* Removed `vcd_independent_disk.size` in favor for `vcd_independent_disk.size_in_mb` [GH-588]
```

Comment:
We cannot just remove `size`, which was not deprecated before. We don't want to cause breakages. Could you deprecate `size` the way you did with `virtual_machine_ids`?

Reply:
Well current size doesn't work well - it's a bug -> https://github.com/vmware/terraform-provider-vcd/issues/355. With latest version you ask one - you will get different thing(size). It's possible to leave this field as deprecated, but only very limited implementation - just for `create` - no read(update of size not possible in general). Result will be unexpected result for user with VCD versions we use.